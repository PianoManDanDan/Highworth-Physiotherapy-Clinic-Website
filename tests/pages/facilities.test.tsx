import React from 'react';
import { getPage } from 'next-page-tester';

describe('Facilities Page', () => {
  test('matches the snapshot', async () => {
    const { render } = await getPage({
      route: '/facilities',
    });

    const page = render();

    expect(page).toMatchSnapshot();
  });
});
