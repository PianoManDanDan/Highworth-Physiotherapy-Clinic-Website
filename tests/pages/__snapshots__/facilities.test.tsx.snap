// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Facilities Page matches the snapshot 1`] = `
Object {
  "nextRoot": <div
    id="__next"
  >
    <header>
      <div
        style="display:inline-block;max-width:100%;overflow:hidden;position:relative;box-sizing:border-box;margin:0"
      >
        <div
          style="box-sizing:border-box;display:block;max-width:100%"
        >
          <img
            alt=""
            aria-hidden="true"
            role="presentation"
            src="data:image/svg+xml;base64,PHN2ZyB3aWR0aD0iMzAiIGhlaWdodD0iMTI4IiB4bWxucz0iaHR0cDovL3d3dy53My5vcmcvMjAwMC9zdmciIHZlcnNpb249IjEuMSIvPg=="
            style="max-width:100%;display:block;margin:0;border:none;padding:0"
          />
        </div>
        <noscript>
          <img
            alt="Highworth Physiotherapy Clinic"
            decoding="async"
            src="/assets/logos/hpc.png"
            srcset="/_next/image?url=%2Fassets%2Flogos%2Fhpc.png&w=32&q=75 1x, /_next/image?url=%2Fassets%2Flogos%2Fhpc.png&w=64&q=75 2x"
            style="position:absolute;top:0;left:0;bottom:0;right:0;box-sizing:border-box;padding:0;border:none;margin:auto;display:block;width:0;height:0;min-width:100%;max-width:100%;min-height:100%;max-height:100%"
          />
        </noscript>
        <img
          alt="Highworth Physiotherapy Clinic"
          decoding="async"
          src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7"
          style="position:absolute;top:0;left:0;bottom:0;right:0;box-sizing:border-box;padding:0;border:none;margin:auto;display:block;width:0;height:0;min-width:100%;max-width:100%;min-height:100%;max-height:100%"
        />
      </div>
      <h1>
        Highworth Physiotherapy Clinic
      </h1>
      <address>
        <h2>
          tel: 
          <a
            href="tel:+441793763814"
          >
            01793 763814
          </a>
        </h2>
      </address>
      <nav>
        <a
          href="/"
        >
          Home
        </a>
        <a
          href="/facilities"
        >
          Facilities
        </a>
        <a
          href="/physiotherapy"
        >
          Physiotherapy
        </a>
        <a
          href="/animals"
        >
          Animals
        </a>
        <a
          href="/meet-the-team"
        >
          Meet the Team
        </a>
        <a
          href="/faq"
        >
          FAQ's
        </a>
        <a
          href="/contact"
        >
          Contact Us
        </a>
      </nav>
    </header>
    <main>
      <h1>
        FACILITIES
      </h1>
      <p>
        We have three fully equipped treatment rooms including one on the ground floor for patients unable to use the stairs.
      </p>
      <p>
        For the convenience of our patients, the clinic also has a separate reception area, waiting room and toilet facilities. We have a small kitchen and can provide refreshments for patients if they are required. At the front of the clinic there is a small shop stocking common rehabilitation items such as: TENS machines; knee, elbow, thumb and wrist braces; Orthaheel insoles and wedges; orthopaedic pillows and glucosamine. We are also happy to order specialist braces for patients if required.
      </p>
      <p>
        Rehabilitation equipment available at the clinic includes: back swing, gym balls, Fitter exerciser, and wobble boards.
      </p>
      <p>
        Treatment equipment available includes: ultrasound, laser, megapulse, infra-red, interferential. The physiotherapists will use the most appropriate equipment for your treatment if they believe it is required.
      </p>
      <p>
        We also operate a free loan system for our patients where we will loan out equipment such as TENS machines, pillows, seat wedges, wobble boards and home traction kits for patients to try before they buy.
      </p>
    </main>
    <footer>
      <div>
        <p>
          Find us on Facebook
        </p>
        <a
          href="https://www.facebook.com/HighworthPhysio/"
          rel="noopener noreferrer"
          target="_blank"
        >
          <div
            style="display:inline-block;max-width:100%;overflow:hidden;position:relative;box-sizing:border-box;margin:0"
          >
            <div
              style="box-sizing:border-box;display:block;max-width:100%"
            >
              <img
                alt=""
                aria-hidden="true"
                role="presentation"
                src="data:image/svg+xml;base64,PHN2ZyB3aWR0aD0iMzIiIGhlaWdodD0iMzIiIHhtbG5zPSJodHRwOi8vd3d3LnczLm9yZy8yMDAwL3N2ZyIgdmVyc2lvbj0iMS4xIi8+"
                style="max-width:100%;display:block;margin:0;border:none;padding:0"
              />
            </div>
            <noscript>
              <img
                alt="Find us on Facebook"
                decoding="async"
                src="/assets/logos/facebook.png"
                srcset="/_next/image?url=%2Fassets%2Flogos%2Ffacebook.png&w=32&q=75 1x, /_next/image?url=%2Fassets%2Flogos%2Ffacebook.png&w=64&q=75 2x"
                style="position:absolute;top:0;left:0;bottom:0;right:0;box-sizing:border-box;padding:0;border:none;margin:auto;display:block;width:0;height:0;min-width:100%;max-width:100%;min-height:100%;max-height:100%"
              />
            </noscript>
            <img
              alt="Find us on Facebook"
              decoding="async"
              src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7"
              style="position:absolute;top:0;left:0;bottom:0;right:0;box-sizing:border-box;padding:0;border:none;margin:auto;display:block;width:0;height:0;min-width:100%;max-width:100%;min-height:100%;max-height:100%"
            />
          </div>
        </a>
      </div>
      <div>
        <p>
          All of our physios are registered with at least one of the following regulatory bodies:
        </p>
        <ul>
          <li>
            <a
              href="https://www.csp.org.uk/"
              rel="noopener noreferrer"
              target="_blank"
            >
              <abbr
                title="Chartered Society of Physiotherapists"
              >
                CSP
              </abbr>
            </a>
          </li>
          <li>
            <a
              href="https://www.hcpc-uk.org/"
              rel="noopener noreferrer"
              target="_blank"
            >
              <abbr
                title="Health and Care Professions Council"
              >
                HCPC
              </abbr>
            </a>
          </li>
          <li>
            <a
              href="https://www.physiofirst.org.uk/"
              rel="noopener noreferrer"
              target="_blank"
            >
              Physio First
            </a>
          </li>
          <li>
            <a
              href="https://www.aacp.org.uk/"
              rel="noopener noreferrer"
              target="_blank"
            >
              <abbr
                title="Acupuncture Association of Chartered Physiotherapists"
              >
                AACP
              </abbr>
            </a>
          </li>
          <li>
            <a
              href="https://www.acpat.org/"
              rel="noopener noreferrer"
              target="_blank"
            >
              <abbr
                title="Association of Chartered Physiotherapists in Animal Therapy"
              >
                ACPAT
              </abbr>
            </a>
          </li>
          <li>
            <a
              href="https://www.rampregister.org/"
              rel="noopener noreferrer"
              target="_blank"
            >
              <abbr
                title="Register of Animal Musculoskeletal Practitioners"
              >
                RAMP
              </abbr>
            </a>
          </li>
        </ul>
      </div>
    </footer>
  </div>,
}
`;
